src/drivers/uart_esc/uart_esc.cpp

```diff
@@ -0,0 +1,563 @@
+/****************************************************************************
+*
+*   Copyright (c) 2015 Mark Charlebois. All rights reserved.
```

Comment:
Is this copyright intentional? Usually it's the PX4 dev group and credit goes in the author field. Same for the other file.